src/SFA.DAS.Assessors.Api.UnitTests/Controllers/WhenGettingTrainingCourseByStandardUId.cs

```diff
@@ -50,14 +50,14 @@ public class WhenGettingTrainingCourseByStandardUId
                     It.IsAny<CancellationToken>()))
                 .Throws<ArgumentException>();
 
-            var controllerResult = await controller.GetByStandardUId(string.Empty) as BadRequestResult;
+            var controllerResult = await controller.GetStandardById(string.Empty) as BadRequestResult;
 
             controllerResult.StatusCode.Should().Be((int)HttpStatusCode.BadRequest);
         }
 
         [Test, MoqAutoData]
         public async Task And_Invalid_StandardUID_Then_Returns_Not_Found(
-            string standardUId,
```

Comment:
Method and class name has StandardUId, can be renamed.

Reply:
Updated